### ISSUE TITLE
Typo in 'kamada_kawai_layout()'

### ISSUE THREAD

**Issue by padix-key** [networkx/networkx#3665]
There is a typo in the docstring  of `kamada_kawai_layout()`. It states that `dist` expects a `float`, but in the text it expects a `dict`:
```
dist : float (default=None)
        A two-level dictionary of optimal distances between nodes,
        indexed by source and destination node.
        If None, the distance is computed using shortest_path_length().
```

**Comment by farhanbhoraniya** [comment]
Created PR for the issue

**Comment by dschult** [comment]
closed in #4059
